server/refishing-fst2.txt

```diff
@@ -0,0 +1,58 @@
+#!/usr/local/bin/foma -f
+# Basic FST for the Germanic dataset
+
+# Dutch
+define Dutch [] ;
+
+clear
+push Dutch
+save stack dutch.bin
+
+# English (Needs fixing)
+def SK "s k " -> "ʃ ";
+def XS "x s " -> "k s ";
+def XH "x " -> "h ";
+def KN "k n " -> "n ";
+
+# hacks
+def AR "a r " -> "ɜ r ";
+def ER "ɜ r " -> "ɜː ";
+
+define English PGWord .o. SK .o. AR .o. ER;
+
+clear
+push English
+save stack english.bin
+
+# Should be able to do this, but it doesn't
+# f ɜː n / f ɜ r n / f a r n
+
+# German
```

Comment:
Please check this line, which you split with `"` on one line, whihc was not ended on the other line, does foma react badly on this?

So you had:

```
"s 
" | ...
```

Reply:
You are totally right, I blindly copied and pasted from your email and the formatting was off. This also helps to fix some problems.